rhizome/host/lib/vm_setup.rb

```diff
@@ -163,7 +163,7 @@ def purge_storage
     storage_roots = []
 
     params = JSON.parse(File.read(vp.prep_json))
-    params["storage_volumes"].each { |params|
+    params["storage_volumes"].reject { _1["read_only"] }.each { |params|
```

Comment:
I know we're pretty unsystematic on rhizome testing, but is there anything cheap-and-easy we can do for now to exercise the read-only code a tiny bit? to make sure it generates the right kind of cloud-hypervisor line. If it's not so straightforward and we need to pencil in "bring rhizome up to speed on test model" for Soon, that's fine too.

Reply:
Added some easy-to-make checks.